Neos.Flow/Classes/Mvc/ActionResponseInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Neos\Flow\Mvc;
+
+use Psr\Http\Message\UriInterface;
+
+interface ActionResponseInterface
+{
+    /**
+     * Set content mime type for this response.
+     *
+     * @param string $contentType
+     * @return void
+     * @api
+     */
+    public function setContentType(string $contentType): void;
+
```

Comment:
I'm missing a `addHeader(string $name, string $value)` method, helping with e.g. setting CORS or cache-control headers in the action.
If we want to elaborate on this, we could maybe design `setAllowHeader()` and `setCacheControl()` methods instead of the 100% generic `addHeader()`.

Reply:
IMO that would be a case of "unplanned extensibility" described in the original [post](https://discuss.neos.io/t/rfc-psr-7-continuation/3597/11)

Reply:
I don't consider setting CORS or caching headers an unplanned extensibility tbqh. It's HTTP basics and highly dependent on the concrete action invoked (this is important, those headers are not global), so moving such decisions to an extra component is bad practice IMO, because it's harder to understand the caching and cross-origin behaviour of an action.